InternetHospital.BusinessLogic/Models/FilteredMyPatientsModel.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace InternetHospital.BusinessLogic.Models
+{
+    public class FilteredMyPatientsModel
```

Comment:
There are generic type for returning pagination, even 2 types: FilteredModel and PageModel.